### DISCUSSION TITLE
Higher blob size not showing in Vercel with error message - "Unable to retrieve the list of blobs"

### DISCUSSION THREAD

**Discussion by dabhishek31** [vercel/community#5927] [Bug Report]
### Page to Investigate

https://vercel.com/[account_name]/[project_name]/stores/blob/[store_name]/browser

### Steps to Reproduce

The file i uploaded was a heavy file approx 330mb. Now I want to delete the file but it's not showing up.
It keeps on throwing the error - 
Unable to retrieve the list of blobs
<img width="675" alt="Screenshot 2024-02-27 at 12 37 14 PM" src="https://github.com/vercel/community/assets/45116121/a72bbf4c-9fe6-49a7-9e75-7216de7164e7">

**Comment by swarnava** [comment]
Hi,

Can you share the store and project url?

**Reply by dabhishek31** [reply]
This is the complete URL

https://vercel.com/dabhishek31/3d-configurator/stores/blob/store_putfgHRgWdNUYoTC/browser
